commands/invite.go

```diff
@@ -14,14 +16,39 @@ func InviteCommands(r *dgc.Router) {
 		Name: "invite",
 
 		Description: "Creates an invite to the server",
+		Usage:       "invite <#channel> <duration> <uses> <reason>",
+		Example:     "invite #welcome 1h30m 0 Single-use for new member",
```

Comment:
this is the most minor thing, but 0 uses would be an infinite invite. i'm not going to request a change for something so minor that's basically just documentation.

Reply:
Yeah that was a mistake on my part. We can change it for sure